dgraph/query.sh

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env bash
 
-curl -s -H "Content-Type: application/graphql+-" "localhost:8080/query" -XPOST --data-binary "@$1" | python3 -m json.tool
\ No newline at end of file
+curl -s -H "Content-Type: application/graphql" "localhost:8080/graphql" -XPOST --data-binary "@$1"
```

Comment:
When running `query.sh` I get `"message": "Unrecognised Content-Type.  Please use application/json for GraphQL requests"`
Changing to `application/json` works in postman, though not when using the script

Reply:
Oops - yes I'm getting the same and did most of my testing via postman, it looks like postman/other graphQL tools do a few modifications behind the scenes before sending the payload off! How misleading - will change the insert.gql to reflect what is actually being sent to the server (required format: https://graphql.dgraph.io/api/)

Reply:
Fixed (and tested via script!) with e4c8c29996f4e8c21348da4fe019cf4232a94de2 (though insertMutation.json looks a little ugly now)

Reply:
Not relevant at this point (but will be later) - seems like in a future official release (it's currently in beta) Dgraph will support Content-Type application/graphql and treat body as graphql request: https://github.com/dgraph-io/dgraph/pull/5125 🥳